contributions/api/jsons/building_blocks_list.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I think we can keep only the list like this:

```
[
  "Admin",
  "App Config",
  "Content",
  "Core",
  "Events",
  "Groups",
  "Learning Management System",
  "Locations",
  "Logs",
  "Notifications",
  "Polls",
  "Rewards",
  "Sports",
  "Talent Chooser",
  "Transportation"
]

```

Reply:
changed